pkg/cmd/pulumi/new.go

```diff
@@ -1083,6 +1090,21 @@ func templatesToOptionArrayAndMap(templates []workspace.Template,
 	return options, nameToTemplateMap
 }
 
+// formatDesc formats the template description nicely if the term width requires it to wrap around.
+func formatDesc(prefixWidth int, maxWidth int, desc string) string {
```

Comment:
minor nit: Consider renaming this function to "reflowDesc" since it's reflowing an already-formatted description.